packages/react-core/src/components/Avatar/examples/Avatar.md

```diff
@@ -37,3 +37,27 @@ import avatarImgDark from './img_avatar-dark.svg';
 <Avatar src={avatarImgDark} alt="avatar" border="dark" />
 ```
 
+### Size variations
+```ts
+import React from 'react';
+import { Avatar } from '@patternfly/react-core';
+import avatarImgDark from './img_avatar-dark.svg';
+
+<React.Fragment>
+  Small
+  <br />
+  <Avatar src={avatarImg} alt="avatar" size="sm" />
```

Comment:
Do you meant to use `avatarImgDark`?  that is what you are importing.  Th build is failing because you are not importing `avatarImg`.  The core docs are using that light image.

Reply:
Thank you @tlabaj, nice catch -- I've updated the path to import the correct image.